libraries/AP_BoardConfig/AP_BoardConfig.cpp

```diff
@@ -340,6 +340,16 @@ const AP_Param::GroupInfo AP_BoardConfig::var_info[] = {
     AP_GROUPINFO("HEAT_LOWMGN", 23, AP_BoardConfig, heater.imu_arming_temperature_margin_low, HAL_IMU_TEMP_MARGIN_LOW_DEFAULT),
 #endif
 
+#if AP_SDCARD_STORAGE_ENABLE
+    // @Param: BRD_SD_MISSION
```

Comment:
I don't understand why we need this param. 64k is tiny. Just allocate 64k and no one will notice.

Reply:
it also costs us 64k of ram, and users need to know their mission can now depend on the microSD, so swapping or formatting the microsd loses the mission